fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1401,6 +1401,16 @@ Future<int64_t> ReadYourWritesTransaction::getEstimatedRangeSizeBytes(const KeyR
 	return map(waitOrError(tr.getStorageMetrics(keys, -1), resetPromise.getFuture()), [](const StorageMetrics& m) { return m.bytes; });
 }
 
+Future<Standalone<VectorRef<KeyRef>>> ReadYourWritesTransaction::getRangeSplitPoints(const KeyRange& range,
+                                                                                     int64_t chunkSize) {
+	if (checkUsedDuringCommit()) {
+		throw used_during_commit();
```

Comment:
why changing to `return`?

Reply:
When returning a `Future` from a function, generally we have the error signaled through the future rather than thrown directly. When inside an `ACTOR`, you can throw an error and the `ACTOR` will put it in the `Future` for you. If you aren't in an `ACTOR`, then throwing the error will actually throw it, while returning it will return a `Future` set with your error.

Reply:
Done.